full-service/src/db/txo.rs

```diff
@@ -645,6 +647,7 @@ pub trait TxoModel {
         max_spendable_value: Option<u64>,
```

Comment:
Good catch, though the way the account/subaddress works is not quite an either/or - the account_id_hex is required, because the wallet supports multiple accounts. But then the subaddress is optional. So for example, as an exchange, I may have Account A (my main account where I custody our exchange funds) and Account B (the account where I custody customer's funds), and then in Account B, I assign subaddresses for my customers to send to, and for the purposes of this PR, at some point I want to spend from the funds received at one of those subaddresses.

Reply:
I think you could potentially make it an either/or using lookups from `assigned_subaddress_b58` -> `account_id`, as that mapping is present in the `AssignedSubaddress` table schema. Though we typically re-derive from `account_keys` -> `subaddress_index` -> `public_address` -> `public_address_b58`.